packages/openneuro-components/src/search-page/FiltersBlock.tsx

```diff
@@ -30,37 +31,73 @@ export const FiltersBlock = ({
   section,
   species,
   domain,
+  selectedDate,
 }: FiltersBlockProps) => {
   const _listItem = (type, item) => {
-    return (
-      <>
-        <li>
-          <strong>{type}:</strong>
-          {type === 'Age' || type === 'Subjects'
-            ? item[0] + ' - ' + item[1]
-            : item}
-          <span>&times;</span>
-        </li>
-      </>
-    )
+    if (item === 'All') {
+      return
+    } else
+      return (
+        <>
+          <li className={type}>
+            <strong>{type}:</strong>
+            <span>
+              {type === 'Age' || type === 'Subjects'
+                ? item[0] + ' - ' + item[1]
+                : item}
+              <span>&times;</span>
+            </span>
+          </li>
+        </>
+      )
   }
```

Comment:
If we do end up using the chiclets for the text list inputs, then i think we should pull this out into its own component. That way we can pass in things like `type` and `item`, as well as a callback for removing that filter from the searchParams.

Reply:
I have another branch where I have updated this so I will do this there, that work?